tests.yaml

```diff
@@ -0,0 +1,12 @@
+# Tests configuration file.
```

Comment:
Can this please be a part of our existing tests that run on LUCI instead of adding a new way of running tests?

Reply:
Yes I will make that change. My current approach was to use this recipe to trigger the tests https://flutter-review.googlesource.com/c/recipes/+/34440

Would this just mean using the existing `.ci.yaml` in flutter engine in this recipe? Or would this be a larger refactor?

Reply:
I'd suggest calling the test script from https://flutter.googlesource.com/recipes/+/refs/heads/main/recipes/engine/engine_lint.py in the "host" case on Linux, if possible.

Reply:
Just updated with the change and removed the other yaml. Thank you for the recommendation.